README.md

```diff
@@ -1,50 +1,95 @@
+| ------------------------------------------------------------------------------------------ | -------------------------------| --------------- |
+| [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-provisioner | 1.14            |
 
-Run a local kubernetes cluster built from latest master code
+## Feature status
 
-## Run Storage Provider
+Various external-provisioner releases come with different alpha / beta features. Alpha features are disabled by default and must be explicitly enabled on command line. Beta features are enabled by default and can be disabled on command line. Check `--help` output for alpha/beta features in each release.
 
-### Use HostPath drivers
+Following table reflects the head of this branch.
 
-Go to drivers and run:
+| Feature    | Status  | Default | Description                                                                                  |
+| ---------- | ------- | ------- | -------------------------------------------------------------------------------------------- |
+| Topology   | Beta    | On      | [Topology aware dynamic provisioning](https://kubernetes-csi.github.io/docs/topology.html)   |
+| Snapshots* | Alpha   | On      | [Snapshots and Restore](https://kubernetes-csi.github.io/docs/snapshot-restore-feature.html) |
 
-```bash
-bin/hostpathplugin --drivername mydriver  --endpoint unix://tmp/csi.sock --nodeid foobar -v=5
-```
+*) There is no special feature gate for snapshots, it is enabled by running [external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter/) and creating its CRDs.
 
-### Start external provisioner
+All other external-provisioner features and the external-provisioner itself is considered GA and fully supported.
 
-```bash
-bin/csi-provisioner --kubeconfig /var/run/kubernetes/admin.kubeconfig --alsologtostderr
-```
+## Usage
 
-### Create Storage class, PVC, and Pod
+It is necessary to create a new service account and give it enough privileges to run the external-provisioner, see `deploy/kubernetes/rbac.yaml`. The provisioner is then deployed as single Deployment as illustrated below:
 
-```bash
-kubectl create -f examples/sc.yaml
-kubectl create -f example/pvc.yaml
-kubectl create -f example/pod.yaml
+```sh
+kubectl create deploy/kubernetes/deployment.yaml
 ```
 
-### Delete PVC
+The external-provisioner may run in the same pod with other external CSI controllers such as the external-attacher, external-snapshotter and/or external-resizer.
 
-```bash
-kubectl delete pvc example-pvc
-```
+Note that the external-provisioner does not scale with more replicas. Only one external-provisioner is elected as leader and running. The others are waiting for the leader to die. They re-elect a new active leader in ~15 seconds after death of the old leader.
+
+### Command line options
+
+#### Important optional arguments that are highly recommended to be used
```

Comment:
Simplify to "Recommended optional arguments"

Reply:
Done